resources/data_model.md

```diff
@@ -0,0 +1,288 @@
+  repositories_environment:          # Optional: add an environment in the repositories path (eg. production, staging) (repositories/PXE)
+
+ep_equipment_type: {server|...}
+
+ep_configuration:
+  keyboard_layout:
+  system_language:
+
+ep_hardware:
+  cpu:
+    architecture:
+    cores:
+    cores_per_socket:
+    sockets:
+    threads_per_core:
+  gpu:
+
+ep_equipment_authentication:
+  user:
+  password:
+```
+
+### Authentication
+
+```yaml
+authentication_root_password_sha512:
+authentication_ssh_keys: []
+```
+
+## To be defined at group_vars/all group level
+
+### General
+
+```yaml
+cluster_name:
+
+time_zone:
+
+icebergs_system: {true|false}
+
+enable_services: {true|false}
+start_services: {true|false}
+
+hosts_file:
+  range: {all|iceberg}
+```
+
+### Network
+
+```yaml
+domain_name:
+
+networks:
+
+  ice1-1:
+    subnet:
+    prefix:
+    netmask:
+    broadcast:
+    dhcp_unknown_range: 10.10.254.1 10.10.254.254
+    gateway:
+    is_in_dhcp: {true|false}
+    is_in_dns: {true|false}
+    services_ip:
+      pxe_ip:
+      dns_ip:
+      repository_ip:
+      authentication_ip:
+      slurm_ip:
+      monitoring_ip:
+      time_ip:
+      log_ip:
+
+  ...
+```
+
+### Security
+
+```yaml
+security:
+  ssh:
+    hostkey_checking: {true|false}
+```
+
+### Repositories
+
+repositories: []
+
+### External integration
+
+```yaml
+external_time:
+  time_server:
+    pool: []
+    server: []
+  time_client:
+    pool: []
+    server: []
+
+external_dns:
+  dns_server: []
+  dns_client: []
+
+external_hosts:
+  hostname: ip
+  ...
+```
+
+### Network File System
+
+```yaml
+nfs_settings:
+  selinux:
+    use_nfs_home_dirs: {true|false}
+
+nfs:
+  softwares:
+    mount:
+    export:
+    server:
+    clients_groups: []
+    take_over_network:
+    export_arguments:
+    mount_arguments:
+  ...
+```
+
+### Internal variables
+
+```yaml
+iceberg_naming:
+equipment_naming:
+management_networks_naming:
+master_groups_naming:
+managements_group_name:
+```
+
+## Advanced usage
+
+### j2_ available variables
+
+```yaml
+j2_master_groups_list: List of master groups.
+j2_equipment_groups_list: List of equipment groups.
+
+j2_icebergs_groups_list: List of icebergs groups.
+j2_number_of_icebergs: Total number of icebergs.
+j2_current_iceberg: Iceberg host is member of. (only one) (icebergX)
+j2_current_iceberg_number: Iceberg number host is member of. (X)
+j2_current_iceberg_network: Iceberg network host is member of (iceX)
+
+j2_node_main_resolution_network: Main resolution network. The network on which host can be ping by direct name. (ex: ping c001).
+j2_node_main_network: Main network. The network used by Ansible to deploy configuration (related to ssh).
+j2_node_main_network_interface: Main network interface. Same as main network, but provides interface name instead of network.
+j2_management_networks: List of management networks.
+
+j2_role_variables: Used in core roles. Sort role vars to grab current OS related ones.
```

Comment:
to remove?

Reply:
Of course !! But I don't get it, why do I do not see it on my local clone ? I even cloned again, and this part of the file is not present in the oxedions/data_model branch. I will do it on github then.